### ISSUE TITLE
what is the second stage IoU do in tensoflow object detection API ?

### ISSUE THREAD

**Issue by Lyhour-Chhay** [tensorflow/models#10147]
Excuse me everyone, in the configuration file for training the model faster RCNN in tensorflow object detection API, there are two IoU: the first stage of IoU is 0.7 and second stage post processing IoU 0.6. I understand what the IoU meaning is. However, I want to know basic of these two IoU in the configuration model faster RCNN. I think the first IoU 0.7 use to get label and for the second IoU what is use for? I don't know about the second IoU. Thank you very much.

**Comment by sachinprasadhs** [comment]
Since, Faster RCNN works in multiple stages.

First IoU is for the Regional Proposed Network ( RPN) to identify the foreground and background anchors based on the IoU threshold.
Second IoU is for the fully connected layer which has a softmax and linear regression layer to classify objects with labels by considering the proposed boxes from RPN and make the adjustment to the bounding boxes near to the accuracy based on the second IoU.
